spec/site/command_spec.rb

```diff
@@ -234,10 +235,18 @@
         timeout: SUPERVISOR_CONFIG['status_update_timeout']
       }
       status = "S0096"
-      expect(result[{"sCI" => status, "n" => "year"}]["s"]).to be == "2020"
-      expect(result[{"sCI" => status, "n" => "month"}]["s"]).to be == "9"
-      expect(result[{"sCI" => status, "n" => "day"}]["s"]).to be == "29"
-      expect(result[{"sCI" => status, "n" => "hour"}]["s"]).to be == "17"
+
+      received = Time.new result[{"sCI" => status, "n" => "year"}]["s"],
+                          result[{"sCI" => status, "n" => "month"}]["s"],
+                          result[{"sCI" => status, "n" => "day"}]["s"],
+                          result[{"sCI" => status, "n" => "hour"}]["s"],
+                          result[{"sCI" => status, "n" => "minute"}]["s"],
+                          result[{"sCI" => status, "n" => "second"}]["s"],
+                          'UTC'
+
+      diff = received - sent
+      expect(diff).to be <= 60
```

Comment:
can we have a situation where diff ends up with a large negative value? if so we need to use diff.abs

Reply:
If the command doesn't work and the clock of the equipment is very off I suppose that could happen. I'll add abs.